<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>903</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="graphBiosignal" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>680</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="graphSound" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>200</y>
      <width>680</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="graphSpectral" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>680</width>
      <height>450</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>740</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
